vernac/g_vernac.mlg

```diff
@@ -83,11 +83,11 @@ GRAMMAR EXTEND Gram
     ]
   ;
   decorated_vernac:
-    [ [ a = attributes ; fv = vernac -> { let (f, v) = fv in (List.append a f, v) }
+    [ [ a = LIST0 quoted_attributes ; fv = vernac -> { let (f, v) = fv in (List.append (List.flatten a) f, v) }
       | fv = vernac -> { fv } ]
```

Comment:
I think this second grammar rule is redundant, as the `a` list may be empty.

Reply:
Changed.